src/constants/index.ts

```diff
@@ -1362,6 +1434,45 @@ export const POOLS_MAP: PoolsMap = {
     route: "tbtc-evmosbtc",
     rewardPids: buildPids({}),
   },
+  [TBTC_METAPOOL_V2_BTCV2_V3_NAME]: {
+    name: TBTC_METAPOOL_V2_BTCV2_V3_NAME,
+    lpToken: TBTC_SWAP_V3_TOKEN,
+    poolTokens: TBTC_POOL_TOKENS,
+    addresses: TBTC_META_SWAP_V3_DEPOSIT_ADDRESSES,
+    isSynthetic: true,
+    type: PoolTypes.BTC,
+    metaSwapAddresses: TBTC_META_SWAP_V3_ADDRESSES,
+    underlyingPoolTokens: TBTC_UNDERLYING_POOL_TOKENS,
+    underlyingPool: BTC_POOL_V2_NAME,
+    route: "tbtcv2btcv2v3",
+    rewardPids: buildPids({ [ChainId.MAINNET]: 5 }),
+  },
+  [WCUSD_METAPOOL_V2_USDV2_V3_NAME]: {
+    name: WCUSD_METAPOOL_V2_USDV2_V3_NAME,
+    lpToken: WCUSD_SWAP_V2_V3_TOKEN,
+    poolTokens: WCUSD_POOL_TOKENS,
+    addresses: WCUSD_META_SWAP_V3_DEPOSIT_ADDRESSES,
+    isSynthetic: true,
+    type: PoolTypes.USD,
+    metaSwapAddresses: WCUSD_META_SWAP_V3_ADDRESSES,
+    underlyingPoolTokens: WCUSD_UNDERLYING_POOL_TOKENS,
+    underlyingPool: STABLECOIN_POOL_V2_NAME,
+    route: "wcusdv2usdv2v3",
+    rewardPids: buildPids({ [ChainId.MAINNET]: 7 }),
+  },
+  [SUSD_METAPOOL_V2_USDV2_V3_NAME]: {
+    name: SUSD_METAPOOL_V2_USDV2_V3_NAME,
+    lpToken: SUSD_SWAP_V2_V3_TOKEN,
+    poolTokens: SUSD_POOL_TOKENS,
+    addresses: SUSD_META_SWAP_V3_DEPOSIT_ADDRESSES,
+    isSynthetic: true,
+    type: PoolTypes.USD,
+    metaSwapAddresses: SUSD_META_SWAP_V3_ADDRESSES,
+    underlyingPoolTokens: SUSD_UNDERLYING_POOL_TOKENS,
+    underlyingPool: STABLECOIN_POOL_V2_NAME,
+    route: "susdv2usdv2v3",
+    rewardPids: buildPids({}),
```

Comment:
double check on chain, but the last pid was 7 and this is new, so I'm guessing this is 9
https://github.com/saddle-finance/multisig-scripts/pull/11/files